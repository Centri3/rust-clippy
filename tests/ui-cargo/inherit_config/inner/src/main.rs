//@compile-flags: --crate-name=inherit_config
#![warn(clippy::many_single_char_names, clippy::too_many_lines)]

fn main() {
    // Inherited from outer config
    let (a, b, c) = (1, 2, 3);
    _ = ();
    _ = ();
    _ = ();
    // Too many lines, not 1 but 3 because of inner config
}
