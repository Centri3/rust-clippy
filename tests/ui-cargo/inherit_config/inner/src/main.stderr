error: 3 bindings with single-character names in scope
  --> $DIR/main.rs:6:10
   |
LL |     let (a, b, c) = (1, 2, 3);
   |          ^  ^  ^
   |
   = note: `-D clippy::many-single-char-names` implied by `-D warnings`

error: this function has too many lines (4/3)
  --> $DIR/main.rs:4:1
   |
LL | / fn main() {
LL | |     // Inherited from outer config
LL | |     let (a, b, c) = (1, 2, 3);
LL | |     _ = ();
...  |
LL | |     // Too many lines, not 1 but 3 because of inner config
LL | | }
   | |_^
   |
   = note: `-D clippy::too-many-lines` implied by `-D warnings`

error: this let-binding has unit value
  --> $DIR/main.rs:7:5
   |
LL |     _ = ();
   |     ^^^^^^ help: omit the `let` binding: `();`
   |
   = note: `-D clippy::let-unit-value` implied by `-D warnings`

error: this let-binding has unit value
  --> $DIR/main.rs:8:5
   |
LL |     _ = ();
   |     ^^^^^^ help: omit the `let` binding: `();`

error: this let-binding has unit value
  --> $DIR/main.rs:9:5
   |
LL |     _ = ();
   |     ^^^^^^ help: omit the `let` binding: `();`

error: aborting due to 5 previous errors

